src/pages/global-info/GlobalInfo.tsx

```diff
@@ -195,28 +146,28 @@ export const GlobalInfo = ({ onSubmit }: { onSubmit: (output: GlobalInfoData) =>
       <h2>Ajoutez des informations pour ces photos </h2>
       <h3>Vous pourrez éditer l&apos;emplacement pour chaque photo à l&#39;étape suivante.</h3>
       <div id="form" className='formBox' >
-        <GlobalInfoForm date={date} onDateTimeChange={onDateTimeChange} valid={valid} onDepartementChange={onDepartementChange}/>
+        <GlobalInfoForm date={date} onDateTimeChange={setDate} onDepartmentChange={setDepartment}/>
       </div>
-      <Checkbox label="J'accepte que ces photos soient intégrées à un jeu de données public" onChecked={onConsentCheckboxCheck} checked={consentCheckboxChecked}/>
+      <Checkbox label="J'accepte que ces photos soient intégrées à un jeu de données public" onChecked={setConsentCheckboxChecked} checked={consentCheckboxChecked}/>
       <Button text='Suivant' filled disabled={!isValid()} onClick={onButtonClick}/>
     </div>
   )
 }
 
 GlobalInfo.displayName = 'GlobalInfo'
 
-export const GlobalInfoForm = ({ date, onDateTimeChange, valid, onDepartementChange, initialDepartement }: { date: Date, onDateTimeChange: (dateTime: Date) => void, valid: { date: boolean, time: boolean }, onDepartementChange: (departement: Value | null) => void, initialDepartement?: string }): JSX.Element => {
+export const GlobalInfoForm = ({ date, onDateTimeChange, onDepartmentChange, initialDepartment }: { date: Date | null, onDateTimeChange: (dateTime: Date) => void, onDepartmentChange: (department: string | null) => void, initialDepartment?: string }): JSX.Element => {
   return (
     <>
-      <DateTimePicker dateTime={date} onChange={onDateTimeChange} valid={{ date: valid.date, time: valid.time }}/>
+      <DateTimePicker dateTime={date} onChange={onDateTimeChange}/>
```

Comment:
Doesn't that break the Datepicker validity check (red underlining when date is invalid) ?